tailwind.config.js

```diff
@@ -11,6 +11,10 @@ module.exports = {
         'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
         'gradient-conic':
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
+          'footerbg': './pngwing 7.png',
+      },
+      screens:{
+        '-sm': {max: '640px'},
```

Comment:
This seems unnecessary as whenever breakpoints are used, the regular styling takes over automatically before those breakpoints are reached.